Lib9c/Action/CombinationEquipment.cs

```diff
@@ -340,9 +333,18 @@ public override IAccountStateDelta Execute(IActionContext context)
             // Transfer Required NCG
             if (costNCG > 0L)
             {
+                var arenaSheet = states.GetSheet<ArenaSheet>();
+                if (arenaSheet.GetRowByBlockIndex(context.BlockIndex) == null)
```

Comment:
is this necessary? i can't find any using row.

Reply:
👍🏻. fixed it. please check again 🏹